src/ClientHandle.h

```diff
@@ -356,7 +356,8 @@ class cClientHandle  // tolua_export
 
 	void HandlePluginMessage    (const AString & a_Channel, const AString & a_Message);
 	void HandleRespawn          (void);
-	void HandleRightClick       (int a_BlockX, int a_BlockY, int a_BlockZ, eBlockFace a_BlockFace, int a_CursorX, int a_CursorY, int a_CursorZ, const cItem & a_HeldItem);
+	void HandleUseItem          (eHand a_Hand);
+	void HandleRightClick       (int a_BlockX, int a_BlockY, int a_BlockZ, eBlockFace a_BlockFace, int a_CursorX, int a_CursorY, int a_CursorZ, eHand a_Hand);
```

Comment:
Please keep these alpha-sorted (and move it in the cpp file to match the order in the header, too)

Reply:
Done.
The cpp file is not well-ordered. I placed it in a relatively better place.
It deserves a dedicated PR if you want the functions in cpp file to be ordered.